core/dropdowndiv.js

```diff
@@ -192,7 +197,23 @@ Blockly.DropDownDiv.getContentDiv = function() {
  */
 Blockly.DropDownDiv.clearContent = function() {
   Blockly.DropDownDiv.content_.innerHTML = '';
-  Blockly.DropDownDiv.content_.style.width = '';
+  Blockly.DropDownDiv.content_.style.cssText = '';
+};
+
+/**
+ * Resets the CSS of the div.
+ * @param {boolean} persistDisplay True to persist the display mode,
+ *    otherwise it will be set to 'none'.
+ */
+Blockly.DropDownDiv.resetCss = function(persistDisplay) {
+  var style = Blockly.DropDownDiv.DIV_.style;
+  var displayMode = style.display;
+  style.cssText = '';
+  style.opacity = 0;
+  style.transition = Blockly.DropDownDiv.TRANSITION;
```

Comment:
Is this just for cleanliness? We weren't setting resetting the transition property.